webapp/components/test/wpt-interop.html

```diff
@@ -73,6 +67,35 @@
           .find(c => c.args.length && c.args[0].pathname === '/api/interop');
       });
     });
+
+    suite('interopScore()', () => {
+      const allFail = [10];
+      const allPass = [10];
+      for (let n = 1; n < 10; n++) {
+        allFail.push(0);
+        allPass.splice(0, 0, 0);
```

Comment:
`unshift` is a nice shortcut for this operation

Reply:
Done.